.github/workflows/deploy.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
```

Comment:
Probably worth getting rid of the code comment as it no longer matches up with what the workflow does.
```suggestion
```